scripts/phar_build.sh

```diff
@@ -6,7 +6,7 @@ echo "Installing composer dependencies with --no-dev..."
 composer install --no-dev
 
 echo "Building terminus.phar..."
-box compile
+"${HOME}"/box/vendor/bin/box compile
```

Comment:
Was this necessary for CI, or did you do this for convenience for local development? I think I'd rather skip this if it's not needed for CI, as it's more restrictive. ideally, there would be a `composer box:install` command for local dev use, but we don't need to do that now.

Reply:
It was breaking in CI.